twoliter/embedded/rpm2img

```diff
@@ -251,81 +249,13 @@ mv "${ROOT_MOUNT}/boot/efi"/* "${EFI_MOUNT}"
 # verification to "just work" later on, regardless of which
 # type of signing profile we have.
 if [ "${UEFI_SECURE_BOOT}" == "yes" ] ; then
-  declare -a SHIM_SIGN_KEY
-  declare -a CODE_SIGN_KEY
-
-  # For an AWS profile, we expect a config file for the PKCS11
-  # helper. Otherwise, there should be a local key and cert.
-  if [ -s "${HOME}/.config/aws-kms-pkcs11/config.json" ] ; then
-    # Set AWS environment variables from build secrets, if present.
-    for var in AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN ; do
-      val="${var,,}"
-      val="${HOME}/.aws/${val//_/-}.env"
-      [ -s "${val}" ] || continue
-      declare -x "${var}=$(cat "${val}")"
-    done
-    # Verify that AWS credentials are functional.
-    aws sts get-caller-identity
-    # Log all PKCS11 helper activity, to simplify debugging.
-    export AWS_KMS_PKCS11_DEBUG=1
-    SB_KEY_SOURCE="aws"
-    SHIM_SIGN_KEY=(-c shim-sign-key -t shim-sign-key)
-    CODE_SIGN_KEY=(-c code-sign-key -t code-sign-key)
+  # shellcheck disable=SC2034  # variables are used externally by reference
+  declare -a SHIM_SIGN_KEY CODE_SIGN_KEY
+  if [[ -s "${HOME}/.config/aws-kms-pkcs11/config.json" ]]; then
+    sbsetup_aws_profile SHIM_SIGN_KEY CODE_SIGN_KEY
```

Comment:
Do we still need `SHIM_SIGN_KEY` and `CODE_SIGN_KEY` to be passed in by reference, or can they be set and saved in `imghelper`?

As a minor nit: we could also move this branch into a single `sbsetup_signing_profile` function that calls the right function (aws or local).